plugins/functions/string.c

```diff
@@ -320,6 +489,13 @@ GRN_PLUGIN_REGISTER(grn_ctx *ctx)
                   func_string_tokenize,
                   NULL, NULL, 0, NULL);
 
+#ifdef GRN_WITH_ONIGMO
```

Comment:
Could you always define `string_slice`?
It's useful without regular expression support.